src/common/TimeUtils.h

```diff
@@ -19,7 +19,14 @@
 #include "MString.h"
 
 #include <time.h>
+
+#ifdef _WIN32
+#include <winsock2.h>
+static BOOL g_first_time = 1;
```

Comment:
@AleksandarZeljic Is this the style we decided on for global variables?

Reply:
@guykatzz There is nothing in the coding guidelines about style of global variables, so I don't think we made a decision. Are you for or against this style? I know we decided to drop the prefix in some cases, where that made sense.

Reply:
SGTM, I can add it to the Coding Guidelines.

Reply:
Pushed a commit to use camel case for the global variables